docs/standard/cross-platform/windowsruntimestreamextensions-asrandomaccessstream-method.md

```diff
@@ -70,5 +70,5 @@ Supported in: Windows 8.1
 
 ## See also
 
--[System.IO.WindowsRuntimeStreamExtensions](https://msdn.microsoft.com/library/system.io.windowsruntimestreamextensions(v=vs.110).aspx)
--[How to: Convert Between .NET Framework Streams and Windows Runtime Streams](../io/how-to-convert-between-dotnet-streams-and-winrt-streams.md)
+- [System.IO.WindowsRuntimeStreamExtensions](https://msdn.microsoft.com/library/system.io.windowsruntimestreamextensions(v=vs.110).aspx)
```

Comment:
This item can be removed; WindowsRuntimeStreamExtensions isn't listed in the System.IO namespace.